### ISSUE TITLE
how to get val_loss on main branch

### ISSUE THREAD

**Issue by Daniel595** [open-mmlab/mmdetection#10267]
Hi there.
First thanks for publishing this great work.

I am succesfully training RTMDet instance segmentation. The coco evaluation is not quite useful for my dataset and I would like to use val_loss. But I can't get it running. 

The solution mentioned in older Issues putting `workflow = [('train', 1),('val', 1)]` to the config doesn't seem to work.

Also I would like to disable coco evaluation.

Can anyone help me out?

Thanks a lot :-)

**Comment by hhaAndroid** [comment]
@Daniel595 Sorry, no support for now

**Comment by Vaaaaaalllll** [comment]
> @Daniel595 Sorry, no support for now

no support yet on getting the validation loss?

**Comment by christianhi123** [comment]
@hhaAndroid Is this planned soon? I think using train loss and validation loss is very common for quick NN analysis. It would be great if validation loss can also be included for overfitting/underfitting determination.
